.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,6 +25,14 @@ I have:
 - [ ] Run `make reviewable` to ensure this PR is ready for review.
 - [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.
 
+### Code review process
+For full details refer to the [contribution process].
+
+- [ ] Request a review from a Reviewer or Code Owner 
+- [ ] At least one approval from a Reviewer or Code Owner
+- [ ] Request a Maintainer to review AFTER approval from a Reviewer/Code Owner
+- [ ] At least one approval from a Maintainer 
```

Comment:
I'm not sure about this check-list. Who fills this out? (The PR author? The folks reviewing the PR?)

* GitHub is setup to automatically request review in the sense that it adds reviewers to the PR.
* I would avoid mentioning Code Owners; technically both reviewers and maintainers are Code Owners.
* GitHub already requires approval from a maintainer.

I do like the idea of including guidance for contributors in this template in order to set expectations around how code review would work, but I would suggest doing so inside a `<!-- -->` style comment and using prose rather than a checklist to:

* Briefly explain the review process.
* Explain to the contributor how to determine which reviewers (as opposed to maintainers) were auto-assigned to their PR, and encourage them to follow up with their reviewers if they don't hear back in a timely manner.
* Link the contributor to the detailed information you've added to `CONTRIBUTING.md`.